### PR TITLE
feat(views): restructured various page elements and improved css

### LINKED ISSUE #14578: Should we introduce <article> and <aside> elements where appropriate?

### LINKED ISSUE #14591: [Bug]: firefox css flickering
### Elgg version

master

### Source of your Elgg installation

GIT clone

### Describe the bug

The screen flickers before the css is applied

### Steps to reproduce

_No response_

### Expected behaviour

_No response_

### Additional notes

_No response_

### PHP version

_No response_

### PHP error log

_No response_

### PR DESCRIPTION
**Pull request by jdalsem**
fixes #14591
fixes #14578 
ref #14452

### CODE REVIEW

File: views/default/admin.css.php [hunk 1]
```diff
@@ -27,31 +26,33 @@
...
 		
-		 .elgg-menu li .elgg-child-menu {
+		.elgg-menu-container {
+			justify-content: space-between;
+			width: 100%;
+		}
+		
+		.elgg-menu li .elgg-child-menu {
 			background: #2d3047;
 		}
```
Comment (jeabakker) [inline views/default/admin.css.php:47]: this could be inside the `.elgg-menu-container` right?

File: views/default/page/elements/owner_block.php [hunk 2]
```diff
@@ -11,39 +11,28 @@
...
-]);
-
-$extra_class = '';
 $body = '';
 if (elgg_extract('show_owner_block_menu', $vars, true)) {
 	$menu_params = elgg_extract('owner_block_menu_params', $vars, []);
 	$menu_params['entity'] = $owner;
 	$menu_params['prepare_vertical'] = true;
-	
+
```
Comment (jeabakker) [inline views/default/page/elements/owner_block.php:24]: you actually didn't touch this

**Review by jeabakker — CHANGES_REQUESTED** [review]

File: mod/profile/views/default/profile/profile.css [hunk 3]
```diff
@@ -32,24 +31,14 @@
...
-		.elgg-layout-sidebar-alt {
-			border-top: none;
-		}
-	}
+}
+
+#profile-details {
+	border: 1px solid $(border-color-soft);
+	background: white;
+	margin-bottom: 1rem;
```
[the remark this answers is not in the document]
Comment (jdalsem) [reply mod/profile/views/default/profile/profile.css:43]: to keep profile details (base + mediaqueries) together

File: views/default/admin.css.php (hunk 1 above)
Comment (jdalsem) [reply views/default/admin.css.php:47]: fixed

File: views/default/page/elements/owner_block.php (hunk 2 above)
Comment (jdalsem) [reply views/default/page/elements/owner_block.php:24]: fixed

**Review by jeabakker — APPROVED** [review]

**Pull request merged**